bump_v1.21.0

```diff
@@ -0,0 +1,41 @@
+diff --git a/Chart.yaml b/Chart.yaml
+index 15c5d9e..6592b03 100644
```

Comment:
What is this diff? What purpose does it serve? 🤔

Reply:
Ooops :) Removed!